odoo/service/db.py

```diff
@@ -64,6 +64,9 @@ def _initialize_db(id, db_name, demo, lang, user_password, login='admin', countr
         with closing(db.cursor()) as cr:
             env = odoo.api.Environment(cr, SUPERUSER_ID, {})
 
+            if phone_number:
+                env.user.partner_id.phone = phone_number
```

Comment:
move to company

Reply:
if country_code:
                 country = env['res.country'].search([('code', 'ilike', country_code)])
                 env['res.company'].browse(1).write({'country_id': country, 'currency_id': country;currency_id, 'phone' :phone:})